src/main/java/com/binance/api/client/domain/account/Order.java

```diff
@@ -237,6 +242,14 @@ public void setOrigQuoteOrderQty(String origQuoteOrderQty) {
     this.origQuoteOrderQty = origQuoteOrderQty;
   }
 
+  public String getOrderListId() {
+    return orderListId;
+  }
+
+  public void setOrderListId(String orderListId) {
+    this.orderListId = orderListId;
+  }
+
   @Override
   public String toString() {
```

Comment:
The `orderListId` field must be added to the `toString` method

```java
.append("orderListId", orderListId)
```

Reply:
Added this one